src/sentry/incidents/__init__.py

```diff
@@ -1,3 +1,4 @@
 from __future__ import absolute_import
 from . import events  # NOQA
-from . import receivers  # NOQA
+
+default_app_config = "sentry.incidents.apps.IncidentsAppConfig"
```

Comment:
The docs say:
> New applications should avoid default_app_config. Instead they should require the dotted path to the appropriate AppConfig subclass to be configured explicitly in INSTALLED_APPS.

So instead of defining this, we could just change what we put in `INSTALLED_APPS` to just point directly to the `AppConfig`

Reply:
Ah, good catch!